src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,170 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(value);
```

Comment:
```suggestion
        Node<T> newNode = new Node<>(tail, value, null);
```
![image](https://github.com/mate-academy/jv-linked-list/assets/20326858/a12eaf94-dad3-4ddf-a26f-c859e73f8da7)